### PR TITLE
Fixes on AmqExternalAccessTest

### PR DESCRIPTION
**Pull request by rimolive**
This is the definitive fix for AmqExternalAccessTest as well as improvements in the other testcases.

### CODE REVIEW

File: amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqMultiReplicasPVTest.java [hunk 1]
```diff
@@ -50,20 +50,18 @@
  * @author Ales Justin
  */
 @RunWith(Arquillian.class)
-// TODO -- change this once multi repl image is in prod
-@Template(url = "https://raw.githubusercontent.com/alesj/application-templates/amq1/amq/amq62-repl.json",
```
Comment (alesj) [inline amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqMultiReplicasPVTest.java]: Leave this template for now, until we get proper template in jboss-openshift repo.

File: amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqMultiReplicasPVTest.java [hunk 2]
```diff
@@ -50,20 +50,18 @@
...
+@Template(url = "https://raw.githubusercontent.com/jboss-openshift/application-templates/master/amq/amq62-persistent.json",
     parameters = {
         @TemplateParameter(name = "MQ_QUEUES", value = "QUEUES.FOO,QUEUES.BAR"),
         @TemplateParameter(name = "MQ_TOPICS", value = "topics.mqtt"),
         @TemplateParameter(name = "APPLICATION_NAME", value = "amq-test"),
+        @TemplateParameter(name = "AMQ_SPLIT", value = "true"),
         @TemplateParameter(name = "MQ_USERNAME", value = "${amq.username:amq-test}"),
         @TemplateParameter(name = "MQ_PASSWORD", value = "${amq.password:redhat}"),
-        @TemplateParameter(name = "MQ_PROTOCOL", value = "openwire,amqp,mqtt,stomp"),
-        @TemplateParameter(name = "IMAGE_STREAM_NAMESPACE", value = "${kubernetes.namespace}")}) // remove when amq-internal-is is removed
```
Comment (alesj) [inline amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqMultiReplicasPVTest.java]: Also leave this -- it's needed in custom stream resource.

**Comment by alesj** [comment]
@rimolive still waiting on reverting the changes from the comments above before merging this ...

**Comment by rimolive** [comment]
I am closing this Pull Request and will send another one containing only the fixes for AmqExternalTest.

**Pull request closed without merging**